content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -123,6 +123,32 @@ const prisma = new PrismaClient({
 })
 ```
 
+### <inlinecode>adapter</inlinecode>
+
+Defines an instance of a driver adapter. See also [Drivers](/concepts/components/drivers) <span class="concept"></span>.
+
+This is available from version 5.4.0 and upwards. 
```

Comment:
Do we usually say "upwards" in these cases? Not "newer" or something more "time-y" instead of "location-y"?

Reply:
I think that is dependent on how you interpret upwards but it can also be used for versioning.